robozilla/bz.py

```
@@ -160,9 +158,24 @@ def get_bug_data(self, bug_id):
                     bug_data = self.set_bug_data_fields(
                         bug, bugs_clones_data=bug_clones_data)
 
-            except (ExpatError, ErrorString, Fault):
-                # to handle this
-                pass
+            except ErrorString as error:
+                logger.warning(
+                    'Could not interpret bug. Error: {0}'.format(error)
+                )
+            except Fault as error:
+                logger.warning(
+                    'Could not fetch bug. Error: {0}'.format(error.faultString)
+                )
+            except ExpatError as error:
+                logger.warning(
+                    'Could not interpret bug. Error: {0}'.format(
+                        ErrorString(error.code)
+                    )
+                )
+            except (TypeError, ValueError):  # pragma: no cover
```

@renzon , think we should keep this raise (by putting  raise after logger.warning), think we need to we need to raise this exception.
@rochacbruno what do you think?